` iter_copied` error

I got following compile error:
```
[170/1118]   Compiling toml v0.5.4
error[E0658]: use of unstable library feature 'iter_copied' (see issue #57127)
   --> /home/dvogt/.cargo/registry/src/github.com-1ecc6299db9ec823/toml-0.5.4/src/de.rs:435:30
    |
435 |                             .copied()
    |                              ^^^^^^

error[E0658]: use of unstable library feature 'iter_copied' (see issue #57127)
   --> /home/dvogt/.cargo/registry/src/github.com-1ecc6299db9ec823/toml-0.5.4/src/de.rs:575:22
    |
575 |                     .copied()
    |                      ^^^^^^

error: aborting due to 2 previous errors

For more information about this error, try `rustc --explain E0658`.
error: Could not compile `toml`.
```
Actually i switch to the 0.4 version.

My version:
```
stable-x86_64-unknown-linux-gnu (default)
rustc 1.35.0 (3c235d560 2019-05-20)
```

--- a/src/de.rs
+++ b/src/de.rs
@@ -432,8 +432,7 @@ impl<'de, 'b> de::MapAccess<'de> for MapVisitor<'de, 'b> {
                         }
                         entries[start..]
                             .iter()
-                            .copied()
-                            .filter(|i| *i < self.max)
+                            .filter_map(|i| if *i < self.max { Some(*i) } else { None })
                             .map(|i| (i, &self.tables[i]))
                             .find(|(_, table)| table.values.is_some())
                             .map(|p| p.0)
@@ -572,8 +571,7 @@ impl<'de, 'b> de::SeqAccess<'de> for MapVisitor<'de, 'b> {
                 }
                 entries[start..]
                     .iter()
-                    .copied()
-                    .filter(|i| *i < self.max)
+                    .filter_map(|i| if *i < self.max { Some(*i) } else { None })
                     .map(|i| (i, &self.tables[i]))
                     .find(|(_, table)| table.array)
                     .map(|p| p.0)

